src/platform/Linux/bluez/Helper.cpp

```diff
@@ -1085,7 +1072,7 @@ static BluezConnection * BluezCharacteristicGetBluezConnection(BluezGattCharacte
 {
     BluezConnection * retval = nullptr;
     GVariantDict * options   = nullptr;
-    GVariant * option_device = nullptr;
+    GAutoPtr<GVariant> option_device;
```

Comment:
Option_device is converted to unique_ptr and later unrefed using raw pointer

Reply:
Yes, I overlooked that, thanks for noticing and review.